hs_core/views/utils.py

```diff
@@ -690,6 +657,7 @@ def unzip_file(user, res_id, zip_with_rel_path, bool_remove_original):
     istorage.session.run("ibun", None, '-xDzip', zip_with_full_path, unzip_path)
     link_irods_folder_to_django(resource, istorage, unzip_path, (zip_fname,))
 
+    # TODO: why was the zipfile part of the django ResourceFile's?
```

Comment:
@alvacouch Addressing you TODO comment, the zip file is part of the django ResourceFile since it is part of the resource to be unzipped. We provide calling function a option whether to remove the original zip file from the resource after it is unzipped.